frontend/src/components/teacher/session-creation/AssessmentsTable.tsx

```diff
@@ -31,6 +39,33 @@ const AssessmentsTable = ({
   setTestName,
   isDisabled = false,
 }: AssessmentsTableProps): React.ReactElement => {
+  const history = useHistory();
+  const { showToast } = useToast();
+
+  const [previewAssessmentQuery] = useLazyQuery<{
+    test: TestResponse;
+  }>(GET_TEST);
+
+  const previewAssessment = async (assessmentId: string) => {
+    const { data } = await previewAssessmentQuery({
```

Comment:
nit: instead of checking for nullish data, we should catch exceptions here, since awaited Apollo lazy queries throw any error instead of returning it